configapp/configapp.py

```diff
@@ -109,6 +155,12 @@ def data_image(filename):
     return send_from_directory(directory, filename)
 
 
+@app.route('/image/captured/<path:img>')
+def captured_image(img):
+    img = img.split('/')
+    return send_from_directory(os.path.join(img_predict_dir, img[0]), img[1])
```

Comment:
@Adman From the PKSpace's point of view, I am sure there is no other request to this endpoint. I am just afraid that this can easily be used to bring the the whole app down, as `GET /image/captured/nonexistent.png` will at the very least result in `IndexError`.

If you'd like to split on `/`, I suggest you at the very least check that `/` is in `img`.